### ISSUE TITLE
invalid usage

### ISSUE THREAD

**Issue by X-dark** [glensc/pbin#1]
It is not explicit that the language detection feature need the file to be put as a parameter and not feed to stdin.

**Comment by glensc** [comment]
yep, should probably refactor it, so if input comes from STDIN the stream is written to temp file, filetype detected and then posted to pastebin.

i've got bitten by this myself too several times.
